lib/scrabble-project/player.rb

```diff
@@ -0,0 +1,32 @@
+module Scrabble
+
+  class Player
+
+    attr_accessor :name, :plays, :total_score
+
+    def initialize(name)
+      @name = name
+      @plays = []
+      @won = false
+      @total_score = 0
+    end
+
+    def play(word)
+      @plays.push(word)
+      return @plays
```

Comment:
Quick style note: Since Ruby returns the last line of a method by default, you don't need to explicitly say `return` here.